tools/src/test/java/org/apache/kafka/tools/consumer/group/DeleteOffsetsConsumerGroupCommandIntegrationTest.java

```diff
@@ -42,109 +56,152 @@
+        }
     }
 
-    @ParameterizedTest
-    @ValueSource(strings = {"zk", "kraft"})
-    public void testDeleteOffsetsOfStableConsumerGroupWithUnknownTopicOnly(String quorum) {
-        testWithStableConsumerGroup("foobar", -1, -1, Errors.UNKNOWN_TOPIC_OR_PARTITION);
+    @ClusterTest
+    public void testDeleteOffsetsOfStableConsumerGroupWithUnknownTopicOnly() {
+        Properties consumerProperties = new Properties();
+        testWithConsumerGroup("foobar", -1, -1, Errors.UNKNOWN_TOPIC_OR_PARTITION, true, consumerProperties);
+        if (clusterInstance.isKRaftTest()) {
+            consumerProperties.put(ConsumerConfig.GROUP_PROTOCOL_CONFIG, GroupProtocol.CONSUMER.name());
+            testWithConsumerGroup("foobar", -1, -1, Errors.UNKNOWN_TOPIC_OR_PARTITION, true, consumerProperties);
+        }
     }
 
-    @ParameterizedTest
-    @ValueSource(strings = {"zk", "kraft"})
-    public void testDeleteOffsetsOfEmptyConsumerGroupWithTopicPartition(String quorum) {
-        testWithEmptyConsumerGroup(TOPIC, 0, 0, Errors.NONE);
+    @ClusterTest
+    public void testDeleteOffsetsOfEmptyConsumerGroupWithTopicPartition() {
+        createTopic(TOPIC);
+        Properties consumerProperties = new Properties();
+        testWithConsumerGroup(TOPIC, 0, 0, Errors.NONE, false, consumerProperties);
+        if (clusterInstance.isKRaftTest()) {
+            consumerProperties.put(ConsumerConfig.GROUP_PROTOCOL_CONFIG, GroupProtocol.CONSUMER.name());
+            testWithConsumerGroup(TOPIC, 0, 0, Errors.NONE, false, consumerProperties);
+        }
     }
 
-    @ParameterizedTest
-    @ValueSource(strings = {"zk", "kraft"})
-    public void testDeleteOffsetsOfEmptyConsumerGroupWithTopicOnly(String quorum) {
-        testWithEmptyConsumerGroup(TOPIC, -1, 0, Errors.NONE);
+    @ClusterTest
+    public void testDeleteOffsetsOfEmptyConsumerGroupWithTopicOnly() {
+        createTopic(TOPIC);
+        Properties consumerProperties = new Properties();
+        testWithConsumerGroup(TOPIC, -1, 0, Errors.NONE, false, consumerProperties);
+        if (clusterInstance.isKRaftTest()) {
+            consumerProperties.put(ConsumerConfig.GROUP_PROTOCOL_CONFIG, GroupProtocol.CONSUMER.name());
+            testWithConsumerGroup(TOPIC, -1, 0, Errors.NONE, false, consumerProperties);
+        }
     }
 
-    @ParameterizedTest
-    @ValueSource(strings = {"zk", "kraft"})
-    public void testDeleteOffsetsOfEmptyConsumerGroupWithUnknownTopicPartition(String quorum) {
-        testWithEmptyConsumerGroup("foobar", 0, 0, Errors.UNKNOWN_TOPIC_OR_PARTITION);
+    @ClusterTest
+    public void testDeleteOffsetsOfEmptyConsumerGroupWithUnknownTopicPartition() {
+        Properties consumerProperties = new Properties();
+        testWithConsumerGroup("foobar", 0, 0, Errors.UNKNOWN_TOPIC_OR_PARTITION, false, consumerProperties);
+        if (clusterInstance.isKRaftTest()) {
+            consumerProperties.put(ConsumerConfig.GROUP_PROTOCOL_CONFIG, GroupProtocol.CONSUMER.name());
```

Comment:
we should test both `CONSUMER` and `CLASSIC` for kraft. Also, could you initialize the consumer configs in construction. for example;
```java
    DeleteOffsetsConsumerGroupCommandIntegrationTest(ClusterInstance clusterInstance) {
        this.clusterInstance = clusterInstance;
        this.consumerConfigs = clusterInstance.isKRaftTest()
            ? Arrays.asList(Collections.singletonMap(ConsumerConfig.GROUP_PROTOCOL_CONFIG, GroupProtocol.CLASSIC.name()),
                Collections.singletonMap(ConsumerConfig.GROUP_PROTOCOL_CONFIG, GroupProtocol.CONSUMER.name()))
            : Collections.emptyList();
    }
```